examples/cornell_box.py

```diff
@@ -229,7 +229,10 @@ def intersect_scene(pos, ray_dir):
 
 @ti.func
 def visible_to_light(pos, ray_dir):
-    a, b, c, mat = intersect_scene(pos, ray_dir)
+    #eps*ray_dir is easy way to prevent rounding error
+    #here is best way to check the float precision:
+    #http://www.pbr-book.org/3ed-2018/Shapes/Managing_Rounding_Error.html
```

Comment:
```suggestion
    # eps*ray_dir is easy way to prevent rounding error
    # here is best way to check the float precision:
    # http://www.pbr-book.org/3ed-2018/Shapes/Managing_Rounding_Error.html
```
Sounds cool! Just some format nits.

Reply:
Here is  where the origin idea from in chinese:
https://forum.taichi.graphics/t/cornell-box-bug/1196/6

I am not familiar with git